app/views/passengers/index.haml

```diff
@@ -0,0 +1,57 @@
+= form_tag(passengers_path, :method => "get") do
+  = label_tag 'Search by name:'
+  = text_field_tag :search, @search_term, placeholder: 'Enter a name'
+  %p
+  = label_tag 'Filter by Type:'
+  Permanent
+  = radio_button_tag :filter, 'permanent'
+  Temporary
+  = radio_button_tag :filter, 'temporary'
+  Show All
+  = radio_button_tag :filter, 'all'
+  = submit_tag "Find passengers"
+%p
+= link_to "Add New Passenger", new_passenger_path
+
+- if @permanent.present?
```

Comment:
They're actually checking for the presence of an instance variable, so this is proper.